File: lab5/echon.pl
#!/usr/bin/perl

$#ARGV+1 == 2 or die "Usage: $0";
$ARGV[0] =~ /^[0-9]+/ or die "Usage: $0";

foreach $a (1..$ARGV[0]){
	print $ARGV[1],"\n";
}

File: lab6/echon.py
#!/usr/bin/python

import sys
import re

if len(sys.argv)!=3:
	sys.exit("Usage: " + sys.argv[0])

if re.search(r"^[0-9]+$",sys.argv[1]) is None:	
	sys.exit("Usage: " + sys.argv[0]);

i = 0;
while i < int(sys.argv[1]):
	print sys.argv[2];
	i = i+1;
